toolkit/smart-contracts/commands/src/governed_map.rs

```diff
@@ -85,6 +90,22 @@ pub struct RemoveCmd {
 	genesis_utxo: UtxoId,
 }
 
+#[derive(Clone, Debug, clap::Parser)]
+pub struct UpdateCmd {
+	#[clap(flatten)]
+	common_arguments: crate::CommonArguments,
+	#[arg(long)]
+	key: String,
+	#[arg(long)]
+	value: ByteString,
+	#[arg(long)]
+	current_value: Option<ByteString>,
```

Comment:
I think `key` and `value` could have a help message, for example about formatting. The first one should be UTF-8 encodable string (with length constrained by the chain runtime settings), the second one should be hex encoded bytes.

This optional parameter for me requires a help message that explains how it affects execution of the command.